packages/reakit/src/Toolbar/__examples__/ToolbarWithPopover/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { useToolbarState, Toolbar, ToolbarItem } from "reakit/Toolbar";
+import {
+  usePopoverState,
+  Popover,
+  PopoverDisclosure,
+  PopoverArrow,
+  PopoverDisclosureProps,
+} from "reakit/Popover";
+import { Button } from "reakit/Button";
+
+const PopoverItem = React.forwardRef<HTMLButtonElement, PopoverDisclosureProps>(
```

Comment:
Hmm, TS is complaining about it. I guess it should be `Partial<PopoverDisclosureProps>` since the required props (e.g. `toggle`) are already being passed by the internal `usePopoverState`.